### PR TITLE
Feature/team gv

### PR DESCRIPTION
**Pull request by GVarenhorst**
This git pull covers: 
1)in team.rb -- class Team, attr_reader, initialize
2)In team_spec.rb -- 'exists',  'has id' and 'has team name'

### CODE REVIEW

File: spec/team_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+require 'rspec'
+require './lib/team'
+
+RSpec.describe Team do
```
Comment (algibson1) [inline spec/team_spec.rb:4]: If you `require 'rspec'` as you do above, you don't need `RSpec.` below. They essentially do the same thing.

That said, does anyone have preference which one to use in all of our specs in order to be consistent? 
Let's discuss at our standup tomorrow

File: lib/team.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require 'csv'
```
Comment (algibson1) [inline lib/team.rb:1]: We can move this `require` to the spec file since `Team` isn't reading any CSV data itself

**Comment by algibson1** [comment]
Looking great!!

File: spec/team_spec.rb (hunk 1 above)
Comment (DChavez18) [reply spec/team_spec.rb:4]: I typically call on RSpec.describe, so I would be in favor of that syntax

**Review by DChavez18 — CHANGES_REQUESTED** [review]

File: spec/team_spec.rb (hunk 1 above)
Comment (GVarenhorst) [reply spec/team_spec.rb:4]: Fixed requested changes

**Review by DChavez18 — APPROVED** [review]

**Pull request closed without merging**